Shortest Path algorithm

```diff
@@ -0,0 +1,44 @@
+#include<iostream>
+#include<iomanip>
+#define NODE 7
+#define INF 999
+using namespace std;
+
+//Cost matrix of the graph
+int costMat[NODE][NODE] = {
```

Comment:
Take input from user and pass array through the function instead using a global variable.